openmetadata-ui/src/main/resources/ui/src/components/Settings/Services/Ingestion/IngestionRecentRun/IngestionRecentRun.test.tsx

```diff
@@ -259,6 +259,35 @@ describe('Test IngestionRecentRun component', () => {
     expect(await screen.findAllByText(/label.log-plural/)).toHaveLength(1);
   });
 
+  it('should show additional details for clicked on non latest run', async () => {
+    (getRunHistoryForPipeline as jest.Mock).mockResolvedValueOnce({
+      data: [...executionRuns],
+      paging: { total: 4 },
+    });
+
+    await act(async () => {
+      render(<IngestionRecentRuns ingestion={mockIngestion} />);
+    });
+
+    const runs = await screen.findAllByTestId('pipeline-status');
+    const partialSuccess = await screen.findByText(/Partial Success/);
+
+    expect(partialSuccess).toBeInTheDocument();
+    expect(runs).toHaveLength(3);
+
+    await act(async () => {
+      // click on second last
+      fireEvent.click(runs[runs.length - 1]);
```

Comment:
This will click on the last run, right?

If there are 4 runs and 4-1=3 that is the last run...

Reply:
Right, So we have tests where we clicked on the Partial Staus, and wanted to check the same with status without label so we used this approach. Let me know if you have any suggestions here

Reply:
Okay, got it, please update the comment `// click on second last`.